flutter/lib/benchmark/resource_manager.dart

```diff
@@ -30,15 +30,18 @@ class BatchPreset {
 
 class ResourceManager {
   static const _zipPattern = '.zip';
-  static const _systemPathPattern = '/sdcard/mlperf_datasets';
+  static const _applicationDirectoryPrefix = 'app://';
   static const _configurationsFileName = 'benchmarksConfigurations.json';
+  static const _jsonResultFileName = 'result.json';
+  static const _loadedResourcesDirName = 'loaded_resources';
 
   final VoidCallback onUpdate;
   final HttpClient _httpClient = HttpClient();
   final BenchmarksConfiguration defaultBenchmarksConfiguration =
       BenchmarksConfiguration(
     'default',
-    'https://raw.githubusercontent.com/mlcommons/mobile_models/main/v1_0/assets/tasks_v2.pbtxt',
+    // [anh] TODO: Replace this temp URL before merge PR
```

Comment:
This still needs to be replaced?

Reply:
Yes. Should have marked this PR as draft.
We need to agreed on this new prefix `app:\\` for local path. Then I will create a new `tasks.pbtxt` file, push it to repo `mobile_models`, replace this link before merge.